jpyutil.py

```diff
@@ -98,18 +98,28 @@ def _add_paths_if_exists(path_list, *paths):
 
 def _get_module_path(name, fail=False, install_path=None):
     """ Find the path to the jpy jni modules. """
-    import imp
-
-    module = imp.find_module(name)
-    if not module and fail:
-        raise RuntimeError("can't find module '" + name + "'")
-    path = module[1]
-    if not path and fail:
-        raise RuntimeError("module '" + name + "' is missing a file path")
-    
+
+    if sys.version_info < (3, 4):
+        import imp
+        details = imp.find_module(name)  # this should raise an ImportError if module is not found
+        if details and fail:
```

Comment:
Shouldn't this be `not details`?

Either way, can we unit test this?

Reply:
It should be `not details`, for sure. I don't think that it will ever get reached, because either imp.find_module() find it and returns something or it raises an ImportError. 

I did test that method on my machine for modules that exist and modules that don't on python2.7, python3.6, and python3.7. You surely mean something specific by unit test. What here?